CONTRIBUTORS.md

```diff
@@ -20,6 +20,7 @@ Contributors to the Chapel 1.25.0 release
 * John Hartman, [HPE]
 * David Iten, [HPE]
 * Aryamaan Jain, individual contributor
+* Ricardo Jesus, individual contributor
```

Comment:
@rj-jesus : If you could check that I've got your name correct, I'd appreciate it!

Reply:
Hi @bradcray, you do, thank you for including it! (even though I don't think I have any major contributions that warrant its addition) If you do decide to include it, would it be possible to change my affiliation to [EPCC](https://www.epcc.ed.ac.uk/) please?

Reply:
We like to give credit for even minor contributions, which we always appreciate (in this case, the improvements to the emacs and vim highlighters), and in hopes of maybe encouraging more.  :)

I'm afraid that we don't typically list affiliations for community contributors whose institutions aren't directly involved in the Chapel project in order to avoid a legal Pandora's box of having an institution claim that someone couldn't contribute code "in their name" (say).  I think the main current outliers to this rule are the GSoC students, who must be registered students to participate in GSoC.   That said, if you've got a webpage that you'd like to link to, whether personal or based at EPCC, I'd be happy make your name hyperlink to that page if that would be satisfactory.

Reply:
Ah ok, understood. Then it's perfect as it is, thank you once again!